lib/index.ts

```diff
@@ -0,0 +1,70 @@
+import { Server, Plugin } from "@hapi/hapi";
+import * as HapiAuthJwt from "hapi-auth-jwt2";
+import verifyJWT, { Validator } from "./verifyJWT";
+import * as keystoresFromFile from "./keystores.json";
+import { assert } from "@hapi/hoek";
+import makeIssueTokenRoute from "./make-issue-token-route";
+
+export type ClientSecrets = {
+  [key: string]: string;
+};
+
+type HapiOidcOptions = {
+  tokenEndpoint?: string;
+  clients?: ClientSecrets;
+  fetchKeystore?: () => object | Promise<object>;
+  validate?: Validator;
+  dev?: boolean;
+  omitCheckExp?: boolean;
```

Comment:
First pass of this plugin makes `omitCheckExp` optional, since many of our apps aren't doing this today. I'd like to promote smooth adoption of this plugin. I'd look to enforce this and remove this option in 3.0